ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/ovn/hypershift/etcd-encryption/create-keys/cucushift-installer-rehearse-azure-ipi-ovn-hypershift-etcd-encryption-create-keys-ref.yaml

```diff
@@ -0,0 +1,21 @@
+ref:
+  as: cucushift-installer-rehearse-azure-ipi-ovn-hypershift-etcd-encryption-create-keys
+  from_image:
+    namespace: ocp
+    name: "4.16"
+    tag: upi-installer
+  timeout: 15m
+  grace_period: 5m
+  commands: cucushift-installer-rehearse-azure-ipi-ovn-hypershift-etcd-encryption-create-keys-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  documentation: |-
+    This step creates a Key Vault in "${SHARED_DIR}/resourcegroup". 
+    It also grants the appropriate permissions on the Key Vault to the Service Principal corresponding 
+    to the current cluster profile. 
+    Next, it creates an active key and a backup key in the Key Vault. 
+    Finally, the URLs of the created keys are written to:
+    - ${SHARED_DIR}/hypershift_azure_etcd_encryption_active_key_url
+    - ${SHARED_DIR}/hypershift_azure_etcd_encryption_backup_key_url
```

Comment:
@jianlinliu  Should we define step in cucushift-installer-rehearse? IIRC, as we discussed earlier, cucushift-installer-rehearse is just a declarative workflow.

Reply:
Good catch, yeah, installer-rehearse is only used to save install provision/deprovision chains and install workflows, not for detailed steps.

Reply:
you can reference some installer similar cases - provision vault key pairs, `azure-provision-disk-encryption-set` and `azure-provision-customer-managed-key`.